TowerNinja/Assets/Scripts/TowerBehaviourScript.cs

```diff
@@ -116,4 +116,8 @@ private void UpdateTowerAppearance()
         if (hpFloored < 1) _levels[0].SetActive(false);
     }
 
+    public int getHealthPoint() {
```

Comment:
I think someone told me this is not needed for the healthbar? I removed it from the tower already